README.md

```diff
@@ -33,7 +33,7 @@ A screenshot of the connected system is shown below:
 ## Our valuable Contributors👩‍💻👨‍💻 :
 | | | | | |
 |:--:|:--:|:--:|:--:|:--:|
-|<a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/ShivaSankeerth"><img src="https://avatars.githubusercontent.com/u/29270279?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>|
+|<a href="https://github.com/Vedant-Jayesh-Oza"><img src="https://avatars.githubusercontent.com/u/75005433?v=4" height="150px" width="150px"></a>| <a href="https://github.com/ShivaSankeerth"><img src="https://avatars.githubusercontent.com/u/29270279?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>|
```

Comment:
```suggestion
|<a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/ShivaSankeerth"><img src="https://avatars.githubusercontent.com/u/29270279?v=4" height="150px" width="150px"></a>| <a href="https://github.com/Vedant-Jayesh-Oza"><img src="https://avatars.githubusercontent.com/u/75005433?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>|
```

No offence but I am a contributor here too T_T

Reply:
Please click on add suggestion to batch and  then merge the commits from GitHub

Reply:
Hey actually I didn't get what you said 😅

Reply:
> Please click on add suggestion to batch and  then commit suggestion from the top on this page only.

Oh okay I removed your name and image and I added mine thats what the problem is ? 😅😅 sorry I didn't knew about this like I thought we need to remove and add ours so sorry 😅😅

Reply:
No issues. Pull the fresh changes from the report and then add yourself.

Reply:
So I need to make one more branch and then pull upstream and then make changes in the code again but adding myself after your name I need to commit changes and make PR right ?

Reply:
Hey I have dm'd you on slack about the issue I am facing please have a look 😅

Reply:
Follow these steps:
- `git pull https://github.com/akshitadixit/VCallZer` to pull all new changes from the repo to your local clone.
- Make your changes, if there are any conflicts, resolve them.
- Commit and push

All these changes will be reflected in the same PR. Reach out for any help.

Reply:
I did same but I am facing issues 😅😅